qubes_menu/desktop_file_manager.py

```diff
@@ -20,11 +20,13 @@
 """
 Helper class that manages all events related to .desktop files.
 """
-import pyinotify
 import logging
 import asyncio
 import os
 import shlex
+
+from watchdog.observers import Observer
```

Comment:
Why this change? In what ways is watchdog better than pyinotify?

Reply:
pyinotify is using `asyncore`, which is deprecated